### ISSUE TITLE
May be integrate VirusTotal check?

### ISSUE THREAD

**Issue by andry81** [adang1345/PythonWin7#4]
Here is a solution through the GitHub Actions pipeline:
https://github.com/crazy-max/ghaction-virustotal

Direct API:
https://github.com/VirusTotal

**Comment by arhadthedev** [comment]
VirusTotal is used to ensure that a third party archive or executable has no malware attached. Why would we need to check our own debug builds recreated anew on every push?

**Comment by adang1345** [comment]
What would be the benefit to this?

**Comment by andry81** [comment]
> What would be the benefit to this?

Already checked for viruses.

**Comment by adang1345** [comment]
I feel as though not much would be gained if I were to integrate a virus check on my side. A user who's really concerned about viruses shouldn't trust virus scan results reported by the uploader of a file because those results could be manipulated. Rather, the user should run his or her own virus scan.

**Comment by andry81** [comment]
That is anyway better than a user check on it's own.

**Comment by adang1345** [comment]
I've decided not to do this. No user should trust virus scan results that I report, so integrating VirusTotal offers minimal security. Users who are concerned about viruses should perform their own scans. Users who are extremely concerned about viruses should not download these files at all. I have included some notes in the repository for how I built these installers, so users that do not trust these files at all can build them on their own machines instead.
